scripts/bb_sim.py

```diff
@@ -266,14 +278,16 @@ def unfinished():
             "highpass",
         )
         lf_acc[:, c] = bwfilter(
-            ampdeamp(
```

Comment:
This one liner becomes hard to follow once split into mulitple lines.

Reply:
i think the typical 
```if statment:
else:
``` 
would probably be more readable